src/isotools/_transcriptome_io.py

```diff
@@ -568,11 +571,11 @@ def _add_chimeric(t, new_chimeric, min_cov, min_exonic_ref_coverage):
                 t.chimeric[new_bp].append(new_chim)
                 for part in new_chim[1]:
                     if part[0] in t.data:
-                        genes_ol = [g for g in t.data[part[0]][part[2][0][0]: part[2][-1][1]] if g.strand == part[1]]
-                        g, _, _ = _find_matching_gene(genes_ol, part[2], min_exonic_ref_coverage)  # take the best - ignore other hits here
-                        if g is not None:
-                            part[4] = g.name
-                            g.data.setdefault('chimeric', {})[new_bp] = t.chimeric[new_bp]
+                        genes_ol = [gene for gene in t.data[part[0]][part[2][0][0]: part[2][-1][1]] if gene.strand == part[1]]
```

Comment:
I suppose `genes_ol` refers to `genes_overlap`? Maybe expand it for all its occurrences in this script?